src/explore-education-statistics-common/src/components/form/FormFieldCheckboxGroup.tsx

```diff
@@ -22,8 +22,8 @@ function FormFieldCheckboxGroup<FormValues>(props: Props<FormValues>) {
           {...field}
           id={id}
           onAllChange={(event, checked) => {
-            if (props.onAllChange) {
-              props.onAllChange(event, checked, options);
+            if (onAllChange) {
+              onAllChange(event, checked, options);
```

Comment:
Don't have to do this, but we could remove the if block and replace it with:

```ts
onAllChange?.(event, checked, options);
```

A similar thing could be done across the rest of this PR as well.